### DISCUSSION TITLE
gunicorn server type exposed in security check when deploying Hue using helm chart

### DISCUSSION THREAD

**Discussion by nikhil-sharma31** [cloudera/hue#3332] [Questions] [answered]
### Description

{HTTP Request Method == POST ;;; HTTP URI == http://k8s-node:nodeport/notebook/api/get_logs;;; HTTP User-Agent == Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36;;; HTTP Return Code == 200;;; HTTP Server Type == gunicorn/19.9.0;;; HTTP Host == k8s-node:nodeport;;; HTTP Response Content Type == application/json Content-Security-Policy: script-src 'self' 'unsafe-inline' 'unsafe-eval' *....

I checked the configuration file in helm chart but it seems there is no way we can stop publishing server type in the response header, please assist

**Comment by Harshg999** [comment]
Hey @nikhil-sharma31, thanks for pointing this out!

@ranade1 @athithyaaselvam - Any pointers/updates on this issue? IIRC this is similar to what you were working on?

**Comment by nikhil-sharma31** [comment]
@Harshg999 , @athithyaaselvam 

Is this resolved? Looks like need to update the lastest version of the helm chart?

**Comment by bjornalm** [comment, marked as answer]
@nikhil-sharma31 
This issue was addressed by @athithyaaselvam in PR https://github.com/cloudera/hue/pull/3222 
It is not available in a release yet but the code is merged so you can pull the latest from master and build if you need it now.
